base_google_map/__manifest__.py

```diff
@@ -0,0 +1,25 @@
+# Copyright (C) 2019, Open Source Integrators
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    'name': 'Base Google Map',
+    'version': '12.0.1.0.0',
+    'author': 'Open Source Integrators, Odoo Community Association (OCA)',
+    'website': 'https://github.com/OCA/geospatial',
+    'license': 'AGPL-3',
+    'category': 'Extra Tools',
+    'depends': [
+        'base_setup',
+        'base_geolocalize',
```

Comment:
Should we also depend on `web_view_google_map` and have it automatically installed.
The module split is for maintainability, and I think the map view should immediately on install.